services/cell-logger/cell-logger

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+
+import dbus
+import re
+import time
+import csv
+
+bus = dbus.SystemBus()
+
+def get_modem_rssi():
+
+    modem_signal_iface = 'org.freedesktop.ModemManager1.Modem.Signal'
+
+    modem_manager_object = bus.get_object('org.freedesktop.ModemManager1', 
+                                          '/org/freedesktop/ModemManager1')
+    modem_manager_iface = dbus.Interface(modem_manager_object, 
+                                         'org.freedesktop.DBus.ObjectManager')
+
+    modem_data = modem_manager_iface.GetManagedObjects()
+    
+    modem_data_str = str(modem_manager_iface.GetManagedObjects())
```

Comment:
Is this variable used anywhere? Remove it if not

Reply:
It was used to store and then print debug information. Now it went unused and will be removed.